libsolidity/formal/SMTChecker.cpp

```diff
@@ -412,6 +412,10 @@ void SMTChecker::endVisit(FunctionCall const& _funCall)
 	case FunctionType::Kind::Internal:
 		inlineFunctionCall(_funCall);
 		break;
+	case FunctionType::Kind::External:
+		resetStateVariables();
+		resetReferences();
```

Comment:
Should be `resetStorageReferences` (even if only storage references have been implemented so far).